R/onLoad.R

```diff
@@ -0,0 +1,5 @@
+.onLoad <- function(libname, pkgname) {
+  if (is.null(getOption("rsconnect.max.bundle.size"))) {
+    options(rsconnect.max.bundle.size = 1048576000)
```

Comment:
For a given user, it is possible to obtain their BundleSizeLimit via an API call.